worlds/saving_princess/__init__.py

```diff
@@ -0,0 +1,123 @@
+from typing import ClassVar, Dict, Any, Type, List
+
+from BaseClasses import Tutorial, ItemClassification as ItemClass
+from Options import PerGameCommonOptions
+from ..AutoWorld import World, WebWorld
```

Comment:
```suggestion
from worlds.AutoWorld import World, WebWorld
```

Imports from outside your world need to be absolute per the specification here: https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/apworld%20specification.md#caveats